jetty-documentation/src/main/asciidoc/distribution-guide/sessions/session-configuration-hazelcast.adoc

```diff
@@ -95,8 +95,8 @@ jetty.session.hazelcast.onlyClient::
 Hazelcast instance will be configured in client mode
 jetty.session.hazelcast.configurationLocation::
 Path to an an Hazelcast xml configuration file
-jetty.session.hazelcast.scavengeZombies::
-True/False. `False` by default. If `true`, jetty will use hazelcast queries to find sessions that are no longer being used on any jetty node and whose expiry time has passed. If you enable this option, and your session stores attributes that reference classes from inside your webapp, or jetty classes, you will need to ensure that these classes are available on each of your hazelcast instances.
+jetty.session.hazelcast.useQueries::
```

Comment:
`killZombies` would have been a much better name... or maybe `slayZombies`.
`useQueries` is a little meaningless... how about `scavengeWithQueries` or `queryForExpired` ?

Reply:
or `queryForOrphans`?

Reply:
Maybe ... I like useQueries because it makes it clear you need to have Hazelcast configured for queries.